src/main/java/com/fa/training/controller/FootballPlayerController.java

```diff
@@ -19,30 +20,85 @@ public FootballPlayerController(FootballPlayerService footballPlayerService) {
         this.footballPlayerService = footballPlayerService;
     }
 
-    @GetMapping(value = "/add-football-player")
-    public String doGetAddFootballPlayer() {
+    @GetMapping(value = "/add-fb")
+    public String addFootballPlayer() {
 
-        FootballPlayerDto footballPlayerDto = new FootballPlayerDto("Phuong", 1000,
+        FootballPlayerDto footballPlayerDto = new FootballPlayerDto("FB_1", 1000,
                 LocalDate.parse("11-11-2021", DateTimeFormatter.ofPattern("dd-MM-yyyy")),
                 LocalDate.parse("11-11-2025", DateTimeFormatter.ofPattern("dd-MM-yyyy")),
-                "14");
+                "01");
 
         footballPlayerService.create(footballPlayerDto);
         return "hello";
     }
 
-    @GetMapping(value = "/get-football-player-by-number-of-shirt")
-    public String doGetGetFootballPlayerByNumberOfShirt() {
+    @GetMapping(value = "/get-fb-number-shirt")
+    public String getFbByNumberShirt() throws DatabaseException {
 
-        FootballPlayerDto footballPlayer = new FootballPlayerDto("Phuong", 1000,
+        FootballPlayerDto fb_1 = new FootballPlayerDto("FB_2", 1000,
                 LocalDate.parse("11-11-2021", DateTimeFormatter.ofPattern("dd-MM-yyyy")),
                 LocalDate.parse("11-11-2025", DateTimeFormatter.ofPattern("dd-MM-yyyy")),
-                "14");
+                "02");
 
-        footballPlayerService.create(footballPlayer);
+        footballPlayerService.create(fb_1);
 
-        FootballPlayerDto fOutput = footballPlayerService.readByNumberOfShirt(footballPlayer.getNumber());
-        System.out.println(fOutput);
+        FootballPlayerDto fb_2 = footballPlayerService.readByNumberOfShirt(fb_1.getNumber());
+        System.out.println(fb_2);
+
+        return "hello";
+    }
+
+    @GetMapping(value = "/get-fb-id")
+    public String getFbById() throws DatabaseException {
+
+        FootballPlayerDto fb_1 = new FootballPlayerDto("FB_3", 1000,
+                LocalDate.parse("11-11-2021", DateTimeFormatter.ofPattern("dd-MM-yyyy")),
```

Comment:
Move this `"dd-MM-yyyy"` to Constants file

Reply:
Thank you, I will fix this line.